components/operations-controller/Makefile

```diff
@@ -1,4 +1,4 @@
-APP_NAME = compass-operations-controller-manager
+APP_NAME = compass-operations-controller
 APP_PATH = components/operations-controller
 BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang-kubebuilder2:v20210216-1.14.0-rc1-535-gb45951c9
```

Comment:
There was some vulnerability in versions of go <=1.15.6 if I recall correctly. This buildpack has [a new version](https://github.com/kyma-project/test-infra/pull/3341) from 4 days ago. As far as I can see it only bumps version of go. Consider using that one instead.